app/controllers/admin/court_application_hearing_controller.rb

```diff
@@ -0,0 +1,30 @@
+module Admin
+  class CourtApplicationHearingController < Admin::ApplicationController
+    def create
+      hearing = Hearing.find(court_application_params[:hearing_id])
```

Comment:
We could simplify and speed up this endpoint by simply doing:

```ruby
CourtApplicationHearing.find_or_create_by!(court_application_params)
redirect_to admin_court_application_path(court_application_params[:court_application_id]), ...
```

The database constraints will protect us from invalid IDs this way, so `find` doesn't do anything for us